lib/commands/updated.js

```diff
@@ -26,8 +26,14 @@ exports.checkUpdatedPackages = function (config) {
 
   var tick = progressBar(packageNames.length);
 
-  var lastTagCommit = execSync("git rev-list --tags --max-count=1");
-  var lastTag       = execSync("git describe " + lastTagCommit);
+  var hasTags = !!execSync("git tags");
```

Comment:
Is this supposed to be `git tag`, since `git tags` isn't a command I don't think